libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4001,7 +4001,11 @@ void GCS_MAVLINK::send_banner()
     // mark the firmware version in the tlog
     const AP_FWVersion &fwver = AP::fwversion();
 
+#ifndef FW_COMMENT
```

Comment:
please use AP_CUSTOM_FIRMWARE_STRING instead

Reply:
I changed it.